client/blocks/plugins-update-manager/schedule-logs.helper.ts

```diff
@@ -18,6 +18,11 @@ export const getLogDetails = ( log: ScheduleLog, plugins: CorePlugin[] ) => {
 		case 'PLUGIN_UPDATES_START':
 			return translate( 'Plugins update started' );
 		case 'PLUGIN_UPDATES_SUCCESS':
+			if ( log.message === 'no_plugins_to_update' ) {
+				return (
+					translate( 'Plugins update completed' ) + ' — ' + translate( 'no plugins to update' )
```

Comment:
I don't know if this sentence construction is suitable for other languages.

I understand you wanted to save one token from translation, but the better approach here is to wrap the whole sentence inside the one translation call.